<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionDialog</class>
 <widget class="QWidget" name="OptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="enableHttpServer">
     <property name="text">
      <string>Support KeypassHttp protocol
This is required for accessing keypass database from ChromeIPass or PassIfox</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="showNotification">
         <property name="text">
          <string>Sh&amp;ow a notification when credentials are requested</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="bestMatchOnly">
         <property name="text">
          <string>&amp;Return only best matching entries for an URL instead
of all entries for the whole domain</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="unlockDatabase">
         <property name="text">
          <string>Re&amp;quest for unlocking the database if it is locked</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="matchUrlScheme">
         <property name="text">
          <string>&amp;Match URL schemes
Only entries with the same scheme (http://, https://, ftp://, ...) are returned</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sortByUsername">
         <property name="text">
          <string>Sort matching entries by &amp;username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sortByTitle">
         <property name="text">
          <string>Sort matching entries by &amp;title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeSharedEncryptionKeys">
         <property name="text">
          <string>R&amp;emove all shared encryption-keys from active database</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeStoredPermissions">
         <property name="text">
          <string>Re&amp;move all stored permissions from entries in active database</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Password generator</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="HttpPasswordGeneratorWidget" name="passwordGenerator" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>Activate the following only, if you know what you are doing!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="alwaysAllowAccess">
         <property name="text">
          <string>Always allow &amp;access to entries</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="alwaysAllowUpdate">
         <property name="text">
          <string>Always allow &amp;updating entries</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="searchInAllDatabases">
         <property name="text">
          <string>Searc&amp;h in all opened databases for matching entries</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Only the selected database has to be connected with a client!</string>
         </property>
         <property name="indent">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="supportKphFields">
         <property name="text">
          <string>&amp;Return also advanced string fields which start with &quot;KPH: &quot;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Automatic creates or updates are not supported for string fields!</string>
         </property>
         <property name="indent">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>HTTP Port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="httpPort">
           <property name="inputMask">
            <string notr="true">d0000</string>
           </property>
           <property name="placeholderText">
            <string>Default port: 19455</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HttpPasswordGeneratorWidget</class>
   <extends>QWidget</extends>
   <header>http/HttpPasswordGeneratorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
